tests/publiccloud/storage_perf.pm

```diff
@@ -99,19 +100,20 @@ sub analyze_previous_series {
         my $previous_records_mean = get_mean_from_db($args, " limit 5 offset 5");
         # we do analysis only if we get back some non-zero values
         if (defined($previous_records_mean) && defined($last_records_mean) && $previous_records_mean != 0 && $last_records_mean != 0) {
-            record_info('ANALYZE', "Checking " . $current_load_type);
-            # this formula detect if mean values differs more than 10%
-            if ((abs($previous_records_mean - $last_records_mean) / $previous_records_mean) * 100 > 10) {
-                die sprintf("Anomaly occurred in http://openqa-perf.qa.suse.de/d/hoRc37HWz/storage-performance?orgId=1&var-os_flavor=%s&var-os_version=%s", $args->{os_flavor}, $args->{os_flavor});
+            my $diff_percents = (abs($previous_records_mean - $last_records_mean) / $previous_records_mean) * 100;
+            record_info('ANALYZE', sprintf("Checking %s:\nFormula: (abs(\$previous_records_mean - \$last_records_mean) / \$previous_records_mean) * 100 = (abs(%s - %s) / %s) * 100 = %s%", $current_load_type, $previous_records_mean, $last_records_mean, $previous_records_mean, $diff_percents));
```

Comment:
IMHO this is way too verbose. We don't need to print out the whole formula. The resulting values and the important values so that you could calculate it yourself should be enough:
```suggestion
            record_info('ANALYZE', "Comparing mean value of 5 datasets\Current dataset: $last_records_mean\nPrevious dataset: $previous_records_mean\nDiff: $diff_percents %");
```

This results in a infobox like 

    Comparing mean value of 5 datasets
    Current dataset: 5
    Previous dataset: 4
    Diff: 20%

Reply:
Why '5' datasets? But yes, I vote for leaving the `Diff` there.